#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_update-config bashisms.dpatch by  <benoit.mortier@opensides.be>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix update-config bashims

@DPATCH@

--- gosa-2.5.19/update-config	2008-01-08 20:01:49.000000000 +0100
+++ gosa-2.5.19/update-config	2009-10-22 21:32:43.000000000 +0200
@@ -8,8 +8,8 @@
 created.
 
 EOF
-read -n1 -sp "Do you want to continue? (y/n)" answer
-echo
+printf "Do you want to continue? (y/n) " "$answer"
+read answer
 
 if [ "$answer" != "y" -a "$answer" != "Y" ]; then
 	echo "Aborted."
